packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/AddBankStatus.tsx

```diff
@@ -0,0 +1,260 @@
+              bankStatus === BankStatusType.BANK_TRANSFER_ACCOUNT_FAILED_INTERNAL ? (
+              <>
+                <FormattedMessage
+                  id='copy.bank_linked_error_rejected_fraud'
+                  defaultMessage='Please try again or select a different payment method.'
+                />
+              </>
+            ) : bankStatus === BankStatusType.BANK_TRANSFER_ACCOUNT_REJECTED ? (
+              <>
+                <FormattedMessage
+                  id='copy.bank_linked_error_account_rejected'
+                  defaultMessage="We believe you have declined linking your account. If this isn't correct, please"
+                />{' '}
+                <Link
+                  size='16px'
+                  weight={500}
+                  target='_blank'
+                  href='https://support.blockchain.com/hc/en-us/requests/new?ticket_form_id=360000190032'
+                >
+                  <FormattedMessage id='copy.contact_us' defaultMessage='contact us' />
+                </Link>
+                .
+              </>
+            ) : (
+              <>
+                <FormattedMessage
+                  id='copy.bank_linked_error'
+                  defaultMessage='Please try linking your bank again. If this keeps happening, please'
+                />{' '}
+                <Link
+                  size='16px'
+                  weight={500}
+                  target='_blank'
+                  href='https://support.blockchain.com/hc/en-us/'
+                >
+                  <FormattedMessage id='buttons.contact_support' defaultMessage='Contact Support' />
+                </Link>
+                .
+              </>
+            )}
+          </Subcontent>
+        </div>
+      </FlyoutContent>
+      <FlyoutFooter collapsed>
+        {bankStatus === BankStatusType.ACTIVE ? (
+          <Button
+            nature='primary'
+            data-e2e='obContinueBankStatus'
+            fullwidth
+            height='48px'
+            onClick={handleClose}
+          >
+            <FormattedMessage id='buttons.continue' defaultMessage='Continue' />
+          </Button>
+        ) : (
+          <>
+            <Button
+              data-e2e='bankLinkTryAgain'
+              height='48px'
+              size='16px'
+              nature='primary'
+              onClick={retryAction}
+              fullwidth
+            >
+              <FormattedMessage id='buttons.tryagain' defaultMessage='Try Again' />
+            </Button>
+            <Button
+              data-e2e='bankLinkCancel'
+              height='48px'
+              size='16px'
+              nature='light'
+              style={{ marginTop: '16px' }}
+              onClick={handleClose}
+              fullwidth
+            >
+              <FormattedMessage id='buttons.cancel_goback' defaultMessage='Cancel & Go Back' />
+            </Button>
+          </>
+        )}
+      </FlyoutFooter>
+    </FlyoutContainer>
```

Comment:
can we move those giant message switch cases/logic to helper functions that return the `FormattedMessage`? will make the component render function a lot more readable

Reply:
this code block is repeated 4 times. anyway to not do that?
```
<Link
  size='16px'
  weight={500}
  target='_blank'
  href='https://support.blockchain.com/hc/en-us/requests/new?ticket_form_id=360000190032'
>
  <FormattedMessage id='copy.contact_us' defaultMessage='contact us' />
</Link>
```